### PR TITLE
Adding some simple examples

### PR DESCRIPTION
**Pull request by yining1023**
I added some simple examples:
- Image classifier with speech output using MobileNet and p5.speech, [demo](https://yining1023.github.io/machine-learning-for-the-web/week5-moreExamples/ImageClassification_VideoSound/).
- Image classifier with translated restuls and speech output using MobileNet, Google Translate API and p5.speech, [demo](https://youtu.be/DmAZ42g7nRQ). (To run this example, you need to replace the google API key in the `sketch.js`)
- Webcam Scavenger Hunt Game using MobileNet, p5.js, p5.speech, [demo](https://yining1023.github.io/machine-learning-for-the-web/week5-moreExamples/ImageClassification_VideoScavengerHunt/).

I will also put the first example to p5 web editor, so it's easier for people to play with it.

### CODE REVIEW

File: p5js/ImageClassification/ImageClassification_VideoScavengerHunt/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ https://opensource.org/licenses/MIT
+-->
+
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Webcam Image Classification with Speech Output using MobileNet, p5.js, p5.speech</title>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.min.js"></script>
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_VideoScavengerHunt/index.html]: We should probably use `0.7.2` unless there is a reason not to?

File: p5js/ImageClassification/ImageClassification_VideoScavengerHunt/index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/addons/p5.dom.min.js"></script>
+  <script src="https://unpkg.com/ml5@0.1.2/dist/ml5.min.js" type="text/javascript"></script>
+  <script src="lib/p5.speech.js"></script>
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_VideoScavengerHunt/index.html]: Is there a CDN for p5 speech?

File: p5js/ImageClassification/ImageClassification_VideoScavengerHunt/sketch.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+const myVoice = new p5.Speech();
+
+function setup() {
+  noCanvas();
+  // Create a camera input
+  video = createCapture(VIDEO);
+  // Initialize the Image Classifier method with MobileNet and the video as the second argument
+  classifier = ml5.imageClassifier('MobileNet', video, modelReady);
+
+  select('#start').mousePressed(() => {
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_VideoScavengerHunt/sketch.js]: I'm very torn about this, but at the moment we aren't using arrow syntax for p5.js examples. I think we might want to change this soon though.

File: p5js/ImageClassification/ImageClassification_VideoScavengerHunt/sketch.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+function classifyVideo() {
+  classifier.predict(gotResult);
+}
+
+// When we get a result
+function gotResult(err, results) {
+  // The results are in an array ordered by probability.
+  const result = results[0].className;
+  const oneWordRes = result.split(',')[0];
+  const top3Res = results.map(r => r.className);
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_VideoScavengerHunt/sketch.js]: These higher order JS array functions (and the `=>` notation) won't be familiar to beginner level p5 users. Maybe it's ok to keep but add a comment with some resources?

File: p5js/ImageClassification/ImageClassification_VideoSoundTranslate/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Webcam Image Classification with Speech Output using MobileNet, p5.js, p5.speech, google translate API</title>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.min.js"></script>
+  <script src="lib/p5.speech.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/addons/p5.dom.min.js"></script>
+  <script src="https://unpkg.com/ml5@0.1.2/dist/ml5.min.js" type="text/javascript"></script>
```
Comment (shiffman) [inline p5js/ImageClassification/ImageClassification_VideoSoundTranslate/index.html]: 0.1.3?

File: p5js/ImageClassification/ImageClassification_VideoScavengerHunt/index.html (hunk 2 above)
Comment (yining1023) [reply p5js/ImageClassification/ImageClassification_VideoScavengerHunt/index.html]: I couldn't find any public link for p5 speech..

File: p5js/ImageClassification/ImageClassification_VideoScavengerHunt/sketch.js (hunk 4 above)
Comment (yining1023) [reply p5js/ImageClassification/ImageClassification_VideoScavengerHunt/sketch.js]: Agree! Added more comments with links.

**Comment by yining1023** [comment]
I just made some updates. It's ready to be reviewed again.

**Comment by shiffman** [comment]
This looks great! Merging!

**Pull request merged**